static/layout.less

```diff
@@ -43,6 +43,14 @@
 }
 #toc-sidebar {
   grid-area: toc-sidebar;
+  overflow: auto;
+  width: 300px;
```

Comment:
The TOC isn’t supposed to have a set width. It’s set dynamically with [fit-content](https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content):

https://github.com/canjs/bit-docs-html-canjs/blob/f5fe51a08ec578aabb60a2119ff3e6f452e38689/static/layout.less#L9